spec/provider/elastic_beanstalk_spec.rb

```diff
@@ -15,11 +15,12 @@
   let(:app) { 'example-app' }
   let(:env) { 'live' }
   let(:bucket_name) { "travis-elasticbeanstalk-test-builds-#{region}" }
+  let(:bucket_path) { "some/app"}
 
   subject :provider do
     described_class.new(
       DummyContext.new, :access_key_id => access_key_id, :secret_access_key => secret_access_key,
-      :region => region, :app => app, :env => env, :bucket_name => bucket_name
+      :region => region, :app => app, :env => env, :bucket_name => bucket_name, :bucket_path => bucket_path
```

Comment:
No. I had made a comment on the PR itself that I wasn't too sure on how to actually go about testing this. I tried digging around through the code to use other providers as examples but came up dry.